### PR TITLE
BG-1336: Donate to ap wallet directly

### PR DESCRIPTION
**Pull request by ap-justin**
Towards BG-1336
## Explanation of the solution
* for unsupported chains, show QR code and address instead
image

* for supported chains, add option to donate directly nonetheless
image

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/App/App.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ const Profile = lazy(() => import("pages/Profile"));
 const Marketplace = lazy(() => import("pages/Marketplace"));
 const Registration = lazy(() => import("pages/Registration"));
 const Donate = lazy(() => import("pages/Donate"));
-const DonateFiatThanks = lazy(() => import("pages/DonateFiatThanks"));
```
Comment (ap-justin) [inline src/App/App.tsx:15]: qr crypto also now redirects to donateFiatThanks, thus rename from donateFiat- to generic donateThanks

**Comment by SovereignAndrey** [comment]
@ap-justin Heads up, looks like a sinlge space needs linting. :sweat_smile: 
```bash
Run yarn lint
./src/constants/ap-wallets.ts format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Formatter would have printed the following content:
  
    3[6](https://github.com/AngelProtocolFinance/angelprotocol-web-app/actions/runs/9210430739/job/25337416801#step:5:7) │ };·
```

**Comment by ap-justin** [comment]
> @ap-justin Heads up, looks like a sinlge space needs linting. 😅
> 
> ```shell
> Run yarn lint
> ./src/constants/ap-wallets.ts format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
> 
>   × Formatter would have printed the following content:
>   
>     3[6](https://github.com/AngelProtocolFinance/angelprotocol-web-app/actions/runs/9210430739/job/25337416801#step:5:7) │ };·
> ```

fixed ser

**Comment by SovereignAndrey** [comment]
@ap-justin This looks good. I think we'll hold off on merging this in until we're sure it will work OK with the user-assigned QR wallet setup (via HD Wallets) that @jp-mariano is working on. The minute it hits staging branch, Operations will be asking, "wen launch?!". :rofl:

**Comment by ap-justin** [comment]
> @ap-justin This looks good. I think we'll hold off on merging this in until we're sure it will work OK with the user-assigned QR wallet setup (via HD Wallets) that @jp-mariano is working on. The minute it hits staging branch, Operations will be asking, "wen launch?!". 🤣

got this ser, temp converting this to draft

**Comment by SovereignAndrey** [comment]
@ap-justin I think we're good to fix the conflicts and do a review to move this to staging now that `v2.3` is live. :+1:

**Comment by ap-justin** [comment]
> @ap-justin I think we're good to fix the conflicts and do a review to move this to staging now that `v2.3` is live. 👍

PR ready for review now ser 🤞

**Review by SovereignAndrey — APPROVED** [review]
LGTM! :rocket: :dog2:

**Pull request merged**